api/v1beta1/hyperconverged_types.go

```diff
@@ -91,6 +91,7 @@ type HyperConvergedSpec struct {
 
 	// ResourceRequirements describes the resource requirements for the operand workloads.
 	// +kubebuilder:default={"vmiCPUAllocationRatio": 10}
+	// +kubebuilder:validation:XValidation:rule="!has(self.vmiCPUAllocationRatio) || self.vmiCPUAllocationRatio != 1",message="Automatic CPU limits are incompatible with a VMI CPU allocation ratio of 1"
```

Comment:
Please add an e2e test for this rule

Reply:
That's what I wrote the e2e test for... Did I get it wrong?